src/zimg/colorspace/gamma.cpp

```diff
@@ -140,6 +140,26 @@ float smpte_240m_inverse_eotf(float x) noexcept
 	return x;
 }
 
+float log10_eotf(float x) noexcept
+{
+	if (x < 0.01f)
```

Comment:
Why is the threshold 0.01f? Similarly, why does 0 map to 0.01f in the inverse? That causes non-monotonic behaviour at the next greatest value.

Reply:
It is defined as such in the specifications. This is for log10 but log316 is similar

```
V=1.0+Log10(Lc )÷2 for1 >= Lc >= 0.01
V=0.0 for0.01>Lc >= 0
```

I mapped 0 -> 0.01 in the inverse because I thought it was the right thing to do, but I was unsure as well.